llvm/utils/TableGen/X86DisassemblerTables.cpp

```diff
@@ -878,7 +879,10 @@ void DisassemblerTables::emitContextTable(raw_ostream &o, unsigned &i) const {
   for (unsigned index = 0; index < ATTR_max; ++index) {
     o.indent(i * 2);
 
-    if ((index & ATTR_EVEX) || (index & ATTR_VEX) || (index & ATTR_VEXL)) {
+    if ((index & ATTR_EVEX) && (index & ATTR_64BIT) &&
+        (index & ATTR_OPSIZE) && (index & ATTR_ADSIZE))
+      o << "IC_EVEX_OPSIZE_ADSIZE";
```

Comment:
It seems you should add code after
```
      if (index & ATTR_OPSIZE)
        o << "_OPSIZE";
```

Reply:
Really? If those attributes are not set for instructions, it seems "()" is always empty string.

Reply:
For here, we are not using instructions to select which context to emit, but just count from 0 to IC_max, so we could get that undefined value.

Reply:
Ah, I see.